src/components/quote/Quote.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import './quote.css';
+import URL_BASE, { API_KEY } from '../../global';
+
+export default function Quote() {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      const res = await fetch(`${URL_BASE}=happiness`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'X-Api-Key': API_KEY,
+        },
+      });
+      const json = await res.json().catch(() => setHasError(true));
```

Comment:
[OPTIONAL] It would be good idea if you used `try..catch` method to catch errors :)